backend/src/users/users.model.ts

```diff
@@ -1,7 +1,14 @@
 export interface Lending {
   userId: number,
+  bookInfoId: number,
+  lendDate: Date,
+  lendingCondition: number,
```

Comment:
[DB](https://www.erdcloud.com/d/6rZ4ae7Ft6CrK83RT)에서 lendingCondition는 number값이 아니라 string값입니다.
```suggestion
  lendingCondition: string,
```

Reply:
수정 완료!